quickref/qsharp-quick-reference.tex

```diff
@@ -782,7 +854,60 @@
     $ R_1(\theta)\ket{1} = e^{i\theta}\ket{1} $  &  \\
 
   \end{keysrefGates}
+  
+  \begin{keysrefMultiGates}{Two-qubit gates}
+  
+    Gate &
+    Matrix Representation & 
+    Ket-Bra Representation &
+    Applying to $\ket{\psi} = \alpha\ket{00} + \beta\ket{01} + \gamma\ket{10} + \delta\ket{11}$ &
+    Applying to basis states & \\
+    
+    CNOT &
+    \usebox\cnotgate & 
+    \usebox\braketcx &
+    $ \text{CNOT}\ket{\psi} = \alpha\ket{00} + \beta\ket{01} + \delta\ket{10} + \gamma\ket{11}$ &
+    $ \text{CNOT}\ket{00} = \ket{00} \newline
+      \text{CNOT}\ket{01} = \ket{01} $ &
+    $ \text{CNOT}\ket{10} = \ket{11} \newline
+      \text{CNOT}\ket{11} = \ket{10} $ \\
+ 
+    SWAP &
+    \usebox\swapgate & 
+    $ \ket{00}\bra{00} + \ket{01}\bra{10} + \ket{10}\bra{01} + \ket{11}\bra{11} $ &
+    $ \text{SWAP}\ket{\psi} = \alpha\ket{00} + \gamma\ket{01} + \beta\ket{10} + \delta\ket{11}$ &
```

Comment:
If we're highlighting the sections of the expression that changed, like in Toffoli gate, could we highlight them here as well?